MapboxNavigation/RouteMapViewController.swift

```diff
@@ -119,27 +119,28 @@ class RouteMapViewController: UIViewController {
         
         muteButton.isSelected = NavigationSettings.shared.muted
         mapView.compassView.isHidden = true
+        
+        mapView.tracksUserCourse = true
+        mapView.updateCourseTracking(location: routeController.location, animated: false)
+        mapView.enableFrameByFrameCourseViewTracking(for: 3)
+        
+        guard !isViewLoaded else { return }
 
         if let camera = pendingCamera {
             mapView.camera = camera
-        } else if let firstCoordinate = route.coordinates?.first {
+        } else if let firstCoordinate = routeController.routeProgress.currentLegProgress.currentStep.coordinates?.first {
             let location = CLLocation(latitude: firstCoordinate.latitude, longitude: firstCoordinate.longitude)
```

Comment:
Can we improve this default case by using the course from first to second coordinate or the course of the location on `RouteController.location` to avoid the initial camera spin?